app/src/types/ChefsSubmissionExport.ts

```diff
@@ -26,14 +26,15 @@ export type ChefsSubmissionFormExport = {
   isIndigenousHousingProviderSupported: boolean;
   isNonProfitSupported: boolean;
   isHousingCooperativeSupported: boolean;
-  latitude: number;
-  longitude: number;
+  latitude: string;
+  longitude: string;
```

Comment:
Is there a reason why we're dropping specificity for these values? Normally iirc lat and long can always be coerced/transformed into absolute number value representations.